app/posts/claim-funding-for-mentors/2024-11-13-round-10-support-console-research-findings.md

```diff
@@ -26,6 +26,12 @@ related:
 
 This research aimed to test the user journeys for payments, sampling and clawbacks within the support console.
 
+## User needs addressed by this research
+
+1. As a support user, I need guidance on how to use the support console for payments, sampling and clawbacks, so that I am able to successfully complete the required tasks.
+
+2. As a support user, I need to know how to action claims, so that I am able to complete tasks payments, sampling and clawbacks tasks relating to claims.
```

Comment:
We have a specific user needs format that you can use here:
```suggestion
{% from "user-need/macro.njk" import appUserNeed %}
{{ appUserNeed({
  description: "As a support user, 
  I need guidance on how to use the support console for payments, sampling and clawbacks,
  So that I am able to successfully complete the required tasks."
}) }}

{{ appUserNeed({
  description: "As a support user, 
  I need to know how to action claims, 
  So that I am able to complete tasks payments, sampling and clawbacks tasks relating to claims."
}) }}
```